requirements.txt

```diff
@@ -0,0 +1,7 @@
+# Base Molo package, pulls in most big things like Wagtail, Django, Pillow
+molo.core == 4.3.2
```

Comment:
we usually don't peg all apps to the same molo core version.. historical.. I'm trying to think of why this would be an issue, but it seems perfectly fine for now.